### PR TITLE
Part three: use new header on chat.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

File: shared/chat/conversation/header-area/normal/index.native.js [hunk 1]
```diff
@@ -61,10 +46,12 @@ const ShhIcon = props => (
 const ChannelHeader = (props: Props) => (
   <Wrapper {...props}>
     <Box2 direction="horizontal" style={styles.channelHeaderContainer}>
-      <Avatar teamname={props.teamName} size={16} />
+      <Avatar teamname={props.teamName} size={12} />
```
Comment (buoyad) [inline shared/chat/conversation/header-area/normal/index.native.js]: should `size` be bigger for the small team header?  I see 16 in some designs
image

File: shared/chat/inbox/row/chat-filter-row/index.js [hunk 2]
```diff
@@ -82,48 +82,63 @@ class ChatFilterRow extends React.PureComponent<Props, State> {
...
-          onBlur={this._stopEditing}
-          onKeyDown={this._onKeyDown}
-          onEnterKeyDown={this._onEnterKeyDown}
-          ref={this._setRef}
-          style={{marginRight: Styles.globalMargins.tiny}}
-        />,
-      ]
+      children = (
+        <Kb.Box style={styles.inputContainer}>
+          <Kb.Icon
```
Comment (buoyad) [inline shared/chat/inbox/row/chat-filter-row/index.js]: You don't need the `key`s on this and the `Input` anymore

**Comment by adamjspooner** [comment]
Thanks, @buoyad! Mind taking another look? (The “new chat” styles were fixed in master, just needed to merge them in.)

**Pull request merged**